pkgs/tools/misc/contacts/default.nix

```diff
@@ -9,15 +9,15 @@ stdenv.mkDerivation rec {
     sha256 = "0wdqc1ndgrdhqapvvgx5xihc750szv08lp91x4l6n0gh59cpxpg3";
   };
 
-  preBuild = ''
-    substituteInPlace Makefile --replace "xcodebuild" "/usr/bin/xcodebuild"
-  '';
+  buildInputs = [ xcbuild Foundation AddressBook ];
 
   installPhase = ''
     mkdir -p $out/bin
-    cp ./build/Deployment/contacts $out/bin
+    cp ./contacts-*/Build/Products/Default-*/contacts $out/bin
   '';
 
+  NIX_LDFLAGS = " -F${Foundation}/Library/Frameworks/ -F${AddressBook}/Library/Frameworks/";
```

Comment:
Sorry, email lost context of thread. I meant that no, Foundation isn't part of stdenv. CoreFoundation is. First of all, very few unix-flavored packages (of the sort we need in the stdenv) use Foundation (more use CoreFoundation, but even that isn't super common), and second of all, it makes the stdenv "unfree" from Hydra's viewpoint, which would be pretty awful 😄